config/locales/en.yml

```diff
@@ -486,3 +486,11 @@ en:
     transparency: See all our transparency data
     written_statement: See all our written statements to Parliament
     authored_article: See all our authored articles
+  broken_links:
+    broken: 
+      title: Broken links
+      subheading: There are broken links in this document
```

Comment:
Would be good to stick with "We’ve found links in this document that may be broken:" here.